generators/generator-base-private.js

```diff
@@ -339,21 +339,31 @@ module.exports = class extends Generator {
      * @returns formatted api description
      */
     formatAsApiDescription(text) {
+        if (!text || text === '') {
```

Comment:
`!text` will cover `text === ''` as well, hence its reduntant (Js truthy covers `false, 0, "", null, undefined, and NaN`)